elements/pfe-jump-links/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-## 1.0.0-prerelease.39 ( TBD )
+## 1.0.0-prerelease.50 ( TBD )
 
-Tag: [1.0.0-prerelease.39](https://github.com/patternfly/patternfly-elements/releases/tag/1.0.0-prerelease.39)
+Tag: [1.0.0-prerelease.50](https://github.com/patternfly/patternfly-elements/releases/tag/1.0.0-prerelease.39)
 
-- [code](url) Init - new component pfe-jump-links
\ No newline at end of file
+- [code](url) Fix - fixed demo file to accurately show the working component
```

Comment:
I love that you updated the component changelog!!  So cool!  We also need to add this to the monorepo changelog for now until I can automate it.